container/go/pkg/utils/utils.go

```diff
@@ -44,3 +49,49 @@ func ReadImage(src, format string) (v1.Image, error) {
 
 	return nil, errors.Errorf("unknown image format %q", format)
 }
+
+// TODO: REMOVE these two functions copied from Winnie's PR, refactor after her PR is merged.
```

Comment:
The functions added in this file are from Winnie's PR, will be removed after her PR gets merged.